tests/phpunit/Unit/Factbox/CachedFactboxTest.php

```diff
@@ -475,6 +483,14 @@ public function outputDataProvider() {
 			->method( 'getContext' )
 			->will( $this->returnValue( new \RequestContext() ) );
 
+		$store = $this->getMockBuilder( '\SMW\Store' )
```

Comment:
Right, but this PR just aims at getting it run with 5.3 (otherwise it would just stop in the middle of a test drive). Those issues should be addressed in follow-up, someday.

I'm not claiming that we do much better but I hope we are not as close as [0].

[0] https://gerrit.wikimedia.org/r/#/c/195297/113/tests/phpunit/includes/auth/AuthManagerTest.php